packages/react/src/breadcrumb-item/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { Link } from "../link";
+import { Text } from "../text";
+import * as styles from "./BreadcrumbItem.css";
+
+export type BreadcrumbItemProps = {
+  followingSeparator?: boolean;
+  href: string;
+  isEllipsis?: boolean;
+  label: string;
+  onClick?: () => void;
+  precedingSeparator?: boolean;
+  separator?: React.ReactNode;
+};
+
+export const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({
+  followingSeparator = false,
+  href,
+  isEllipsis = false,
+  label,
+  onClick,
+  precedingSeparator = true,
+  separator = ">",
+}) => (
```

Comment:
I think we don't follow this convention. 

```suggestion
export const BreadcrumbItem= ({
  followingSeparator = false,
  href,
  isEllipsis = false,
  label,
  onClick,
  precedingSeparator = true,
  separator = ">",
}: BreadcrumbItemProps ) => (
```

if we need to allow ref, we need to use higher order component.